### PR TITLE
Automerge the daily Dependabot + ZoneDB PRs

### PR DESCRIPTION
**Pull request by case**
Our goal here: Auto-merge the Dependabot-created PRs (like [this one](https://github.com/domainr/whoistest/pull/23)), which update the `github.com/zonedb/zonedb` version each day.

#### Todo

- [x] Try to get it working with the default GH token
- [ ] Constrain it to the Dependabot -> ZoneDB dep bumps, somehow (see below)
- [ ] Constrain it to the `github.com/zonedb/zonedb` line in `go.mod` (if possible)

### CODE REVIEW

**Review by ydnar — APPROVED** [review]
Nice! 

Might want to rename this file (.yaml).

File: .github/workflows/automerge.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+#   pull_request:
+#     types:
+#       - labeled
+#   check_suite:
+#     types:
+#       - completed
+
+# jobs:
+#   automerge:
+#     runs-on: ubuntu-latest
```
Comment (ydnar) [inline .github/workflows/automerge.yaml:22]: These three lines are still needed. The `if` directive is within a job.

**Review by ydnar — CHANGES_REQUESTED** [review]

File: .github/workflows/automerge.yaml (hunk 1 above)
Comment (case) [reply .github/workflows/automerge.yaml]: 👍

**Comment by case** [comment]
Ok, have another look!

**Review by ydnar — APPROVED** [review]

**Comment by ydnar** [comment]
Do we only want to automerge `zonedb` changes?

**Comment by case** [comment]
Yeah, ideally we would constrain this further to just the ZoneDB dep.

**Comment by case** [comment]
There might be something in here we can use?

https://docs.github.com/en/free-pro-team@latest/actions/reference/context-and-expression-syntax-for-github-actions#github-context

Examples:

- The PR names look like this: "Bump github.com/zonedb/zonedb from 1.0.2794 to 1.0.2795"
- Branch names look like this: "`dependabot` merged 1 commit into `main` from `dependabot/go_modules/github.com/zonedb/zonedb-1.0.2795`"
- The PR description text starts with: "Bumps github.com/zonedb/zonedb from 1.0.2794 to 1.0.2795."

**Comment by case** [comment]
Re: fetching some PR metadata we can use: https://github.community/t/how-to-get-pr-branch-name-in-github-actions/16598/6

> this produced the branch to be merged as expected:
> `github.event.pull_request.head.ref`

Or:

> `${{ github.event.pull_request.title }}`

**Comment by case** [comment]
But… what we'd ideally be checking for is that the `github.com/zonedb/zonedb` line in `go.mod` is what changed.

**Pull request merged**